library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss

```diff
@@ -599,3 +599,13 @@ table.table-bordered.dataTable {
     background:#fff;
     padding:0 10px;
 }
+
+label {
+	font-weight: bold;
+}
```

Comment:
i'm curious whether we took this out purposefully

Reply:
It's hard to remember. It could have been me.